src/tests/remote.test.ts

```diff
@@ -15,6 +16,35 @@ const launchOptions = {
 	},
 };
 
+function setupDebugger(
+	conn: perlDebuggerConnection,
+	file: string,
+	cwd: string,
+	args: string[],
+	launchOptions: LaunchOptions
+): [Promise<RequestResponse>, LocalSession] {
+
+	// Not to conflict with VS Code jest ext
+	const port = 5000 + Math.round(Math.random()*100);
```

Comment:
is random needed?

Reply:
I just moved the code, you probably had your reasons to use `random` there. With the changes from another PR it might be possible to use `0` as port so an available port is chosen automatically.

Reply:
So that's why it's random :)
Sounds good about 0!